### ISSUE TITLE
buildables remains do not lay on floor after explosion

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#1059]
Using 0.51.0 assets on 0.51.0 game/engine, when a buildable explodes (either alien or human), we can see the debris flying in the air but they not lay on the floor. Perhaps it's a z index issue and they are just laying under the floor, I don't know.

This bug is not blocking for 0.51.0, it does not look at expected, but it does not look weird either (they just disappear too quickly), but we must not forget it's a bug.

**Comment by illwieckz** [comment]
See this log:

```
Debug: R_AddMDVSurfaces: no such frame -2147483648 to -2147483648 for 'models/fx/metal_gibs/m_gib3.md3' (1) 
```

I get similar logs for md3 rifle shells, and iirc they are not laying on the floor too.

**Comment by Viech** [comment]
What else is there left of MD3 files? Can we not nuke the format already? 💥

**Comment by illwieckz** [comment]
There is some md3 third-person view weapon models still remaining in repositories, I don't know if they are still needed or not.

**Comment by Viech** [comment]
It should be. Buildable remains are just particles after all.

**Comment by illwieckz** [comment]
So this is issue is not an issue, right?

**Comment by Viech** [comment]
As long as they bounce if they are allowed to by cgame, no. Note that bouncing and colliding are probably interchangable terms here as the collision check is the certainly the costy thing about making particles bounce (or stop).

I guess the long-term solution here would be to specify bounciness on a per-particle level (if this is not already the case?), use this only for particles where this makes sense given the performance cost (like buildable remains), and set the default value of `cg_bounceParticles` back to `1`.

**Comment by illwieckz** [comment]
Viech, is your “no” a confirmation for “it's not an issue” affirmation or an invalidation for “right?” question?

In other words, can I close this issue? :smile:

**Comment by Viech** [comment]
Should be closed by #1070 in either case. :stuck_out_tongue:
